approval_polls/templates/approval_polls/edit.html

```diff
@@ -119,15 +119,15 @@ <h4 class='panel-title'>
               <div class='radio'>
                 <label class='full-width'><input type='radio' name='radio-poll-type' {% if poll.vtype == 3 %} checked='true' {% endif %} value='3'>Only specific Email owners (will be invited by Email).
                   <div id='email-input' hidden>
-                    <h6>Enter comma-separated Email IDs:</h6>
+                    <h6>Enter a comma after each email address:</h6>
```

Comment:
@rupalkhilari I changed the messaging to prevent the issue of the vote invitation not getting sent if there is only 1 email address entered and not tabbed. It doesn't get tokenized properly in that case. If you think that might not be an issue, I will stick to the previous messaging. If you are ok with it, then I can change the text at other places that you have pointed out.